Zend/Zend.m4

```diff
@@ -353,6 +353,9 @@ if test "$ZEND_GCC_GLOBAL_REGS" != "no"; then
 #if defined(__GNUC__) && ZEND_GCC_VERSION >= 4008 && defined(i386)
 # define ZEND_VM_FP_GLOBAL_REG "%esi"
 # define ZEND_VM_IP_GLOBAL_REG "%edi"
+#elif defined(__GNUC__) && ZEND_GCC_VERSION >= 4008 && defined(__aarch64__)
+# define ZEND_VM_FP_GLOBAL_REG "x27"
+# define ZEND_VM_IP_GLOBAL_REG "x28"
```

Comment:
I'd suggest moving this down to the bottom of the list. It's a bit weird to have aarch64 registers between x86 and x86_64.

Reply:
^.^ Happy to get your advice. I have move them to the bottom of the list.